recipes/imagej/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "imagej" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 800fbc9f0bc22d97d856daf8eef7aebee969fa8b22523aa59060eab44a305ec4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vvv"
```

Should we install with `--no-deps` flag?
```suggestion
  script: "{{ PYTHON }} -m pip install . -vvv  --no-deps"
```